lib/credly/actions/badge_templates.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Credly
+  module Actions
+    module BadgeTemplates
+      def badge_templates_get(params = {})
+        connection.get("organizations/#{organization_id}/badge_templates", params)
+      end
+
+      def badge_templates_get_single(badge_template_id)
```

Comment:
Instead of `get_single`, how would you feel about doing `badge_templates_list` for getting the list of all BTs, and then `badge_template_get` for getting the single BT? It's pretty common to use plural and singular entity names in these cases.